### PR TITLE
C51-317:  Add empty state for new cases block

### PR DESCRIPTION
**Pull request by AkA84**
This PR adds a new `<civicase-panel-query-empty>` slot that allows the customization of the "empty state" of the panel in case there are no results to display

This is what the empty state looks like by default if the slot is not filled
empty-default

This is the customized empty state for the new cases block
empty-case
```html
<civicase-panel-query>
  <!-- ... -->
  <panel-query-empty>
    <div class="civicase__activity-card--empty">
      <div class="civicase__activity-no-result-icon civicase__activity-no-result-icon--case"></div>
      <div class="civicase__activity-card--big--empty-title">No new {{customData.itemName || 'items'}}</div>
      <div class="civicase__activity-card--big--empty-description">
        You have no new cases this {{selectedRange}}
      </div>
    </div>
  </panel-query-empty>
</civicase-panel-query>
```

## Comments
Given the deadline, the `civicase__activity-card--empty` component was reused, with some additional styles specific to panel-query added on top of it, only for layout purposes. The component is not really semantic in this context, but it provides the desired style out of the box

### CODE REVIEW

File: ang/civicase/PanelQuery.html [hunk 1]
```diff
@@ -13,53 +13,45 @@ <h2 class="panel-title pull-left">
...
-        text-next="{{ ts('Next') }}"
-        text-prev="{{ ts('Previous') }}"
-      ></paging>
-      <div ng-if="loading.full">
-        Showing - to - of - {{customData.itemName || 'items'}}
-      </div>
-      <div ng-if="!loading.full">
-        Showing {{pagination.range.from}} to {{pagination.range.to}} of {{total}} {{customData.itemName || 'items'}}
-      </div>
+    <paging class="pull-right" page="pagination.page" page-size="pagination.size"
```
Comment (swastikpareek) [inline ang/civicase/PanelQuery.html]: Can we add it in multiple lines ?

File: scss/components/_activity-card.scss [hunk 2]
```diff
@@ -286,6 +286,10 @@
   background-image: url('../resources/icons/activities.svg');
 }
 
+.civicase__activity-no-result-icon--case {
```
Comment (swastikpareek) [inline scss/components/_activity-card.scss]: This is not related to activity so it should not be `.civicase__activity-no-result-icon--case`. In `_contact-cases.scss` https://github.com/compucorp/uk.co.compucorp.civicase/blob/develop/scss/components/_contact-cases-tab.scss#L43 
we already use `url('../resources/icons/cases.svg');`. 

Can we have a more generic class which can be reused at both the places?

Comment (AkA84) [reply scss/components/_activity-card.scss]: was agreed that the class name is correct given the circumstance, and to delete the the other class
